bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -77,38 +87,67 @@
+            <h2>{{ _('Firefox <span>for</span> desktop') }}</h2>
+            <p>{{ _('Trusted. Flexible. Fast.') }}</p>
+          </a>
+        </div>
       </li>
 
-      <li class="product highlight" id="product-android">
-        <a href="{{ url('firefox.android.index') }}" data-product="Firefox for Android">
-          {# L10n: span below adds visual style to "for" text #}
-          <h2>{{ _('Firefox <span>for</span> Android') }}</h2>
-          <p>{{ _('Make it uniquely yours.') }}</p>
-        </a>
+      <li class="product {% if not waffle.switch('firefox-ios-global') %}highlight{% endif %}" id="product-android">
+        <div class="inner">
+          <a href="{{ url('firefox.android.index') }}" data-product="Firefox for Android">
+            {# L10n: span below adds visual style to "for" text #}
+            <h2>{{ _('Firefox <span>for</span> Android') }}</h2>
+            <p>{{ _('Make it uniquely yours.') }}</p>
+          </a>
+
+        {% if not waffle.switch('firefox-ios-global') %}
+          <div class="btn-google-play">
+            <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-product="Firefox for Android, Google Play">
+              {{ high_res_img('firefox/android/btn-google-play.png', {'alt': _('Get it on Google Play'), 'width': '129', 'height': '45', 'l10n': True}) }}
+            </a>
+            <small class="download-other">
+              <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+            </small>
+          </div>
+        {% endif %}
+        </div>
+      </li>
 
-        <div class="btn-google-play">
-          <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" data-product="Firefox for Android, Google Play">
-            {{ high_res_img('firefox/android/btn-google-play.png', {'alt': _('Get it on Google Play'), 'width': '129', 'height': '45', 'l10n': True}) }}
+    {% if l10n_has_tag('tracking_protection') and waffle.switch('firefox-ios-global') %}
+      <li class="product highlight" id="product-ios">
+        <div class="inner">
+          <a href="{{ url('firefox.ios') }}" data-product="Firefox for iOS">
+            {# L10n: <span> tag below adds visual style to "for" text #}
+            <h2>{{ _('Firefox <span>for</span> iOS') }}</h2>
+          {% if l10n_has_tag('ios_product_tagline') %}
+            <p>{{ _('Fast. Smart. Yours.') }}</p>
+          {% else %}
+            <p>{{ _('Get it on iPhones and iPads') }}</p>
+          {% endif %}
           </a>
-          <small class="download-other">
-            <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
-          </small>
-        <div>
+
+          <a rel="external" href="{{ settings.APPLE_APPSTORE_FIREFOX_LINK }}" class="btn-apple-appstore" data-product="Firefox for iOS, App Store">
+            <img src="{{ l10n_img('firefox/ios/btn-apple-appstore.svg') }}" alt="{{ _('Download on the App Store') }}" width="135" height="40">
```

Comment:
Name of the button here should be `{{ l10n_img('firefox/ios/btn-app-store.svg') }}` - will resolve in a follow on PR